Mathlib/Data/Stream/Init.lean

```diff
@@ -20,6 +20,7 @@ namespace Stream'
 
 universe u v w
 variable {α : Type u} {β : Type v} {δ : Type w}
+variable (m n : ℕ) (x y : List α) (a b : Stream' α)
```

Comment:
There are plenty of statements in this file that use a different order of these variables. Therefore I don't think putting them at the top is a good idea. Rather, I would specifically use `variable` for certain sections where all statements use them in the same order and explicitness. (like before `get_append_left` on line 453)